reference/configuration/framework.rst

```diff
@@ -55,7 +55,7 @@ Configuration
 * `session`_
     * `storage_id`_
     * `handler_id`_
-    * `name`_
+    * :ref:`name <session-name>`
```

Comment:
Too bad, this means that the anchor used currently (http://symfony.com/doc/current/reference/configuration/framework.html#name) won't work in the future. Maybe @wouterj has an idea how we can prevent the loss.